low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -289,12 +289,12 @@ class SSPLDevDeploy:
 
```

Comment:
I am not able to add comment on particular line, so commenting here:
line no: 240
please update self.RMQ_PASSWD in sspl_tests.conf file also. We need this for sanity.

Reply:
Done.